backend/app/models/question.rb

```diff
@@ -20,6 +20,9 @@ class Question < ApplicationRecord
 
     all.includes([:options])
   }
+  def answered?
```

Comment:
Add just a blank line here. 😬 
```suggestion

  def answered?
```